src/CoreText/CTLine.cs

```diff
@@ -233,7 +233,7 @@ public nfloat GetOffsetForStringIndex (nint charIndex)
 		}
 
 		public delegate void CaretEdgeEnumerator (double offset, nint charIndex, bool leadingEdge, ref bool stop);
-		unsafe delegate void CaretEdgeEnumeratorProxy (IntPtr block, double offset, nint charIndex, bool leadingEdge, ref bool stop);
+		unsafe delegate void CaretEdgeEnumeratorProxy (IntPtr block, double offset, nint charIndex, [MarshalAs (UnmanagedType.I1)] bool leadingEdge, [MarshalAs (UnmanagedType.I1)] ref bool stop);
```

Comment:
I had it in 3 places initially - then dropping the _extra_ ones. I'll monitor the bots to be sure my local (and this build) are not just lucky.